database/seedFunctions.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable no-await-in-loop */
+const axios = require('axios');
+
+const generateRandomPercentage = () => (Math.floor(Math.random() * 100) / 100);
+
+const generateNumberWithinRange = (min, max) => (Math.floor(Math.random() * (max - min) + min));
+
+const generateFillerText = async (options) => {
+  let text;
+  if (options.paras) {
+    text = await axios.get(`https://baconipsum.com/api/?type=meat-and-filler&paras=${options.paras}&format=text`);
+  } else if (options.sentences) {
+    text = await axios.get(`https://baconipsum.com/api/?type=meat-and-filler&sentences=${options.sentences}&format=text`);
+  }
+  return text.data;
+};
+
+const generateLanguageList = () => {
+  const chosenLanguages = [];
+  const languages = [
+    'Arabic',
+    'French',
+    'Portuguese (European)',
+    'Chinese (Simplified)',
+    'Italian',
+    'Vietnamese',
+    'German',
+    'Russian',
+    'English',
+    'Hebrew',
+    'Spanish',
+    'Hindi',
+    'Japanese',
+    'Turkish',
+    'Gujarati',
+    'Polish',
+    'Persian',
+    'Kannada',
+    'Romanian',
+  ];
+  const numberOfLangsToUse = generateNumberWithinRange(0, languages.length);
+  for (let i = 0; i < numberOfLangsToUse; i++) {
+    const randomLanguageIndex = generateNumberWithinRange(0, languages.length);
+    chosenLanguages.push(languages.splice(randomLanguageIndex, 1)[0]);
```

Comment:
Nice logic here but is removal of languages from the language list necessary? 

Would `chosenLanguages.push(languages[randomLanguageIndex])` suffice ?

Reply:
Hmm good point! I initially was thinking I didn't want multiples of the same language in the list, thus the removals. I suppose I could have also just selected a random index range, like from 0-3 or from 2-9 and sliced the whole thing, that would reduce the time complexity.

Reply:
Changed - modified based on suggestion but took a slightly different route. Still, reduced down to constant time.